src/video-token-server.js

```diff
@@ -34,7 +34,7 @@ module.exports.handler = (context, event, callback) => {
     return;
   }
 
-  if (API_PASSCODE + appID !== passcode) {
+  if (API_PASSCODE + appID + serverlessID !== passcode.replace(/\s+/g, '')) {
```

Comment:
Ok sounds good. I was just trying to account for a potential issue. If a web user ever copies and pastes the passcode (with spaces) into the web app's login page, when they won't be able to use the app. They would have to remove the spaces before it would work. 

I removed this change.